test/JDBC/upgrade/14_7/schedule

```diff
@@ -356,6 +356,7 @@ sys-table_types_internal
 sys-table_types_internal-dep
 BABEL-CHECK-CONSTRAINT
 BABEL-3640
+openjson
```

Comment:
BABEL-3702 should be here as well.

Reply:
Actually, this will make 14.7 to latest test failure until 2.x.dev PR is merged.